content/recipes/findability/findingCompounds.md

```diff
@@ -0,0 +1,80 @@
+# Finding FAIR Experimental Datasets for Chemical Compounds
+ "Using Wikidata to find experimental data related chemical compounds in a FAIR dataset."
+
+
+ ````{panels_fairplus}
+:identifier_text: FCB053 
+:identifier_link: 'https://w3id.org/faircookbook/FCB053'
+:difficulty_level: 2
+:recipe_type: hands_on
+:reading_time_minutes: 15
+:intended_audience: bioinformatician, data_curator, data_scientist, data_engineer
+:has_executable_code: nope
+:recipe_name: How to find FAIR Datasets
+```` 
+
+## Main Objectives
+
+The goal of this recipe is to find compounds from scientific sources to add to Wikidata. Many different trusted sources and repositories contain research surrounding chemical compounds, and, given the proper licenses, these compounds can be added to Wikidata.
+___
+
+## Dataset Extensions and Identifiers
+
+Here are some datasets types which are simple to add to Wikidata. Each has a well-structured format. 
+
+* SDF file (FairSharing doi:[10.25504/fairsharing.ew26v7](https://doi.org/10.25504/fairsharing.ew26v7))
+
+* csv file (FairSharing [bsg-s001546/](https://fairsharing.org/bsg-s001546/))
```

Comment:
Actually, we should also mention https://jcheminf.biomedcentral.com/articles/10.1186/s13321-021-00520-4 here. I already created a FAIRsharing entry for that, but since the website is down for maintenance, I cannot look up the identifier right now.

Reply:
Note to self: Cite the article in the comment above. Refer to the other recipes that cite articles

Reply:
Add article to this BibTeX file: https://github.com/FAIRplus/the-fair-cookbook/blob/migrating/_bibliography/bibliography-faircookbook.bib

Reply:
Here's how to add references, using `footcite`, see line 53: https://github.com/FAIRplus/the-fair-cookbook/blame/migrating/content/recipes/infrastructure/chemical-identities.md#L53